yt_dlp/extractor/pbs.py

```diff
@@ -187,9 +193,9 @@ class PBSIE(InfoExtractor):
     _VALID_URL = r'''(?x)https?://
         (?:
            # Direct video URL
-           (?:%s)/(?:(?:vir|port)alplayer|video)/(?P<id>[0-9]+)(?:[?/]|$) |
+           (?:%s)/(?!show)(?:(?:vir|port)alplayer|video)/(?P<id>[^/]+)(?:[?/]|$) |
```

Comment:
The addition to the regex here makes it conflict with the regex from PBSShowIE further below. Also the new regex is not used at all in the tests for this extractor (further down in line 207)…?

Reply:
> makes it conflict with the regex from PBSShowIE further below

Not sure I understand. My thinking here was to exclude any PBS url with /show to have it resolve to PBSShowIE instead

Reply:
Ah. Yes, of course. Sorry, I overlooked the `?!`.